requirements.txt

```diff
@@ -1,6 +1 @@
-git+https://github.com/m-labs/migen@master#egg=migen
-git+https://github.com/m-labs/misoc@master#egg=misoc
-git+https://github.com/peteut/migen-axi@master#egg=migen-axi
-pydantic>=1.5.1
-paramiko>=2.7.2
-scp>=0.14.1
+-e .
```

Comment:
Should we not just remove the rquirements file? This is just a hack with no real benefits and unexpected behaviour (as it also install this package if you run `pip install -r requirements.txt`

Reply:
Later I would use a pyproject.toml